design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -136,6 +136,10 @@ type Mutation {
         destination_id: ID, @doc(description: "unique Id of destination requisition list") # If null new requisition list will be created
         item_ids: [ID!]! @doc(description: "selected requisition list items that are to be moved from source")
     ): MoveItemsFromRequisitionListOutput @doc(description: "Move Items from Requisition List to another requisition List")
+
+    clearCustomerCart(
+	cart_id: String! @doc(description: "masked Cart Id")
```

Comment:
I would also appreciate if you could fix other argument names in this document to be lower camel case instead of underscore. This is currently enforced with static test during the implementation.

```suggestion
	cartId: String! @doc(description: "Cart ID")
```

Reply:
@paliarush 
Sure will change the arguments to camel case.
Is the schema changes fine?

Reply:
These are the only concerns I have, the rest looks good to me.